content/concur/_index.en.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Concur
+---
+
+## Consensus is Not One-Size-Fits-All
+One of the things we wanted to communicate with the Yellowbrick project is that there is no one “best” machine learning model, only a set of best practices for finding the best model for a given dataset. We believe the distributed systems community has a similar problem; everyone is looking for the golden consensus algorithm, but there’s no silver bullet. What makes sense for Google isn’t best for Facebook; what’s used at Twitter isn’t ideal for Netflix. Different systems need different consensus solutions.
```

Comment:
@KNGJOEL Can you please add in the link to the Yellowbrick project that I had above? 

https://www.scikit-yb.org/en/develop/

Reply:
Yes, sorry, It is done now.